service/test/_registry_test.js

```diff
@@ -680,6 +680,114 @@ describe('Registry', function () {
     });
   });
 
+  describe('#importDevicesFromBlobByIdentity', function () {
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_001: [A `ReferenceError` shall be thrown if `inputBlobContainerUri` is falsy]*/
+    [undefined, null, ''].forEach(function(badUri) {
+      it('throws a ReferenceError if inputBlobContainerUri is \'' + badUri + '\'', function() {
+        var registry = new Registry({ host: 'host', sharedAccessSignature: 'sas' });
+        assert.throws(function(){
+          registry.importDevicesFromBlobByIdentity(badUri, 'outputContainerUri', function() {});
+        }, ReferenceError);
+      });
+    });
+
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_002: [A `ReferenceError` shall be thrown if `outputBlobContainerUri` is falsy]*/
+    [undefined, null, ''].forEach(function(badUri) {
+      it('throws a ReferenceError if outputBlobContainerUri is \'' + badUri + '\'', function() {
+        var registry = new Registry({ host: 'host', sharedAccessSignature: 'sas' });
+        assert.throws(function(){
+          registry.importDevicesFromBlobByIdentity('inputContainerUri', badUri, function() {});
+        }, ReferenceError);
+      });
+    });
+
+    testErrorCallback('importDevicesFromBlobByIdentity', 'input', 'output');
+
+    /*Tests_SRS_NODE_IOTHUB_REGISTRY_16_031: [The `importDeviceFromBlob` method shall construct an HTTP request using information supplied by the caller, as follows:
+    ```
+    POST /jobs/create?api-version=<version> HTTP/1.1
+    Authorization: <config.sharedAccessSignature>
+    Content-Type: application/json; charset=utf-8
+    Request-Id: <guid>
+
+    {
+      "type": "import",
+      "inputBlobContainerUri": "<input container Uri given as parameter>",
+      "outputBlobContainerUri": "<output container Uri given as parameter>"
+    }
```

Comment:
This spec doesn't actually say that you have a storage authentication property.  That would be nice.